src/App.css

```diff
@@ -36,7 +36,6 @@
     flex: 1;
```

Comment:
I tried to launch this app locally and got this error 

ERROR in ./src/components/SubHeading/SubHeading.jsx 5:0-26

"Module not found: Error: Cannot find file: 'Subheading.css' does not match the corresponding name on disk: './src/components/SubHeading/SubHeading.css'."

Are we aware of this?